src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-add-node.ts

```diff
@@ -1,654 +1,731 @@
+import "./add-node/zwave-js-add-node-searching-devices";
+import "./add-node/zwave-js-add-node-select-security-strategy";
+import "./add-node/zwave-js-add-node-failed";
+import "./add-node/zwave-js-add-node-configure-device";
+import "./add-node/zwave-js-add-node-code-input";
+import "./add-node/zwave-js-add-node-loading";
+import "./add-node/zwave-js-add-node-added-insecure";
+import "./add-node/zwave-js-add-node-grant-security-classes";
+import { slugify } from "../../../../../common/string/slugify";
+import { computeStateName } from "../../../../../common/entity/compute_state_name";
+import { updateEntityRegistryEntry } from "../../../../../data/entity_registry";
+import type { EntityRegistryEntry } from "../../../../../data/entity_registry";
+import { fullEntitiesContext } from "../../../../../data/context";
+
+const INCLUSION_TIMEOUT_MINUTES = 5;
 
 @customElement("dialog-zwave_js-add-node")
 class DialogZWaveJSAddNode extends LitElement {
+  // #region variables
   @property({ attribute: false }) public hass!: HomeAssistant;
 
-  @state() private _params?: ZWaveJSAddNodeDialogParams;
+  @state() private _open = false;
+
+  @state() private _step?: ZWaveJSAddNodeStage;
 
   @state() private _entryId?: string;
 
-  @state() private _status?:
-    | "loading"
-    | "started"
-    | "started_specific"
-    | "choose_strategy"
-    | "qr_scan"
-    | "interviewing"
-    | "failed"
-    | "timed_out"
-    | "finished"
-    | "provisioned"
-    | "validate_dsk_enter_pin"
-    | "grant_security_classes"
-    | "waiting_for_device";
+  @state() private _controllerSupportsLongRange = false;
 
-  @state() private _device?: ZWaveJSAddNodeDevice;
+  @state() private _supportsSmartStart?: boolean;
 
-  @state() private _stages?: string[];
+  @state() private _dsk?: string;
+
+  @state() private _dskPin = "";
+
+  @state() private _error?: string;
 
   @state() private _inclusionStrategy?: InclusionStrategy;
 
-  @state() private _dsk?: string;
+  @state() private _lowSecurity = false;
 
-  @state() private _error?: string;
+  @state() private _lowSecurityReason?: number;
+
+  @state() private _device?: ZWaveJSAddNodeDevice;
+
+  @state() private _deviceOptions?: ZWaveJSAddNodeSmartStartOptions;
 
   @state() private _requestedGrant?: RequestedGrant;
 
   @state() private _securityClasses: SecurityClass[] = [];
 
-  @state() private _lowSecurity = false;
+  @state() private _codeInput = "";
 
-  @state() private _lowSecurityReason?: number;
+  @query("ha-dialog") private _dialog?: HaDialog;
 
-  @state() private _supportsSmartStart?: boolean;
+  private _qrProcessing = false;
 
   private _addNodeTimeoutHandle?: number;
 
+  private _onStop?: () => void;
+
   private _subscribed?: Promise<UnsubscribeFunc | undefined>;
 
-  private _qrProcessing = false;
+  private _newDeviceSubscription?: Promise<UnsubscribeFunc | undefined>;
+
+  @consume({ context: fullEntitiesContext, subscribe: true })
+  _entities!: EntityRegistryEntry[];
```

Comment:
Does this work in a dialog? As it is provided by the config panel I think the order of the elements is not correct for this to work? 🤔

Reply:
It worked when I tested it 🤷

Reply:
I can test it on my system again just to be sure

Reply:
It wasn't working. I reverted it and fixed the entity naming :+1: